_src/static/sass/utilities/_variables.scss

```diff
@@ -362,6 +363,12 @@ $headings-primary-color: palette(grayscale, x-dark);
 $headings-secondary-font-family: $font-family-sans-serif;
 $headings-secondary-color: palette(grayscale, base);
 
+// buttons
+$buttons-border-style: solid;
+$buttons-border-size: 1px;
+$buttons-border-radius: 3px;
+$buttons-shading: 3px;
```

Comment:
Also, this is less shading than it is a shadow. Let's make that connection more direct with a ``.btn-shadow`` (or ``.btn-elevated-shadow`` if you agree with my previous comment).